src/score-calculator.ts

```diff
@@ -1,5 +1,18 @@
-import { BaseMetric, BaseMetricValue } from './models';
-import { parseMetricsAsMap } from './parser';
+import {
+  AnyMetric,
+  BaseMetric,
+  BaseMetricValue,
+  EnvironmentalMetric,
+  environmentalMetrics,
+  EnvironmentalMetricValue,
+  Metric,
+  MetricValue,
+  metricsIndex,
+  ScoreResult,
+  TemporalMetric,
+  temporalMetrics,
```

Comment:
Maybe think of another name for `temporalMetrics` it might be difficult to differentiate the two variables in the code with just a plural and case difference.

Reply:
environmentalMetricMap would match better i suppose:

```
  EnvironmentalMetric,
  environmentalMetricMap,
  EnvironmentalMetricValue,
```